src/components/layout/Banner.js

```diff
@@ -1,23 +1,188 @@
+      );
+      this.setState({ wrapperHeight: maxElemHeight });
+    }
+  };
+
+  componentWillUnmount = () => {
+    this.timer && clearInterval(this.timer);
+  };
+
+  render() {
+    const { slides } = this.props;
+    if (!slides || slides.length === 0) {
+      return null;
+    } else {
+      return (
+        <Wrapper>
+          <InnerWrapper height={this.state.wrapperHeight}>
+            {this.multipleSlides() && (
+              <CircleWrapper>
+                {this.state.slides.map((_, index) => (
+                  <Circle
+                    key={index}
+                    active={index === this.state.currentBanner}
+                    onClick={() => {
+                      this.setState({
+                        currentBanner: index,
+                      });
+                      this.resetTimerTick();
+                    }}
+                  />
+                ))}
+              </CircleWrapper>
+            )}
+            {this.state.slides.map((elem, index) => {
+              let icon;
+              if (elem.icon) {
+                try {
+                  icon = require(`${elem.icon}`);
+                } catch (err) {
+                  console.error(err);
+                }
+              }
+              return (
+                <ContentWrapper
+                  multipleSlides={this.multipleSlides()}
+                  innerRef={divElement =>
+                    this.wrapper
+                      ? (this.wrapper[index] = divElement)
+                      : (this.wrapper = [divElement])
+                  }
+                  active={index === this.state.currentBanner}
+                  key={index}
+                >
+                  {elem.icon && icon && <Icon src={icon} alt="Banner Icon" />}
+                  {elem.text ? (
+                    elem.url ? (
+                      <Link
+                        href={elem.url}
+                        target={elem.external ? "_blank" : "_self"}
+                      >
+                        {elem.text}
+                      </Link>
+                    ) : (
+                      <Text>{elem.text}</Text>
+                    )
+                  ) : (
+                    <Text>{"Provide valid text for banner"}</Text>
+                  )}
+                </ContentWrapper>
+              );
+            })}
+          </InnerWrapper>
+        </Wrapper>
+      );
+    }
+  }
+}
+
+export default Banner;
 
 const Wrapper = styled.section`
   box-shadow: 0 2px 10px 0 rgba(0, 0, 0, 0.2);
-  background-color: #0b74de;
+  background-color: ${colors.blue};
+  display: flex;
+  justify-content: center;
+  align-items: center;
 `;
 
-const BannerLink = styled.a`
-  color: #fff;
-  text-decoration: underline;
+const InnerWrapper = styled.div`
+  padding: 0 30px;
+  @media (max-width: 736px) {
+    padding: 0 15px;
+  }
+  ${props => props.height && `min-height: ${props.height}px;`};
+  width: 1200px;
```

Comment:
Remove fixed width

Reply:
`1200px` is not a random value, it's the "site's width", look into `global.css` > `.container`

Reply:
It is still fixed. Do not put a fixed value in `width` property. You can set up fixed `max-width`, but not `width.
So the correct values are:
```suggestion
  width: 100%;
  max-width: 1200px;
```